modules/lazy-images/lazy-images.php

```diff
@@ -108,6 +108,20 @@ static function process_image( $matches ) {
 			unset( $new_attributes['sizes'] );
 		}
 
+		/**
+		 * Allow plugins and themes to override the attributes on the image before the content is updated.
+		 *
+		 * One potential use of this filter is for themes that set `height:auto` on the `img` tag.
+		 * With this filter, the theme could get the width and height attributes from the
+		 * $new_attributes array and then add a style tag that sets those values as well, which could
+		 * minimize reflow as images load.
+		 *
```

Comment:
Could you add `@module lazy-images` here so it gets caught in the parser?

Thanks!